mozilla/fx-private-relay
# MPP-3132 - Create a news item for S&P mailing list

This PR fixes MPP-3132

L10N PR: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/150 
  


# New feature description

As a Firefox Relay user I would like to be able to learn more about privacy and security from a trusted source so I can be protected online. Create news item on the firefox relay website

Figma: https://www.figma.com/file/S8hMIBWphwqDmikvTb3O2T/News-UI-Templates?type=design&node-id=22-113 

# Screenshot (if applicable)

![image](https://github.com/mozilla/fx-private-relay/assets/3924990/ad98ea34-227e-4a64-99a1-422673e95077)

# How to test 

Check your news items - there should be a new item.

Content:

Headline options:

Get help protecting your privacy

Body:

It's not easy to keep up with the latest privacy and security tips and habits. So we started a newsletter to help you make your online life more secure.

CTA: Subscribe to the newsletter

# Checklist (Definition of Done)
